src/pages/app-development/starter-kit/hooks.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Webhooks example
+description: Learn how to use webhooks with Adobe Commerce Extensibility Starter Kit.
+keywords:
+ - Extensibility
+ - App Builder
+ - API Mesh
+ - Events
+ - REST
+ - Tools
+---
+
+# Webhooks example
+
+The following runtime action demonstrates how to expose an entry point to perform a realtime check of the available stock for Commerce cart items. You must implement the stock-checking logic on an external service.
```

Comment:
I find the statement `The following runtime action` a bit misleading as there is no `runtime action` after this statement. Maybe it could be rephrased to something like:
> The runtime action at `actions/webhook/check-stock/index.js` demonstrates ...